### PR TITLE
Hometask #5: Modules and services

### PR DESCRIPTION
**Pull request by MarharytaKrasikava**
- Delete course service added
- Authentication service added

### CODE REVIEW

File: src/app/shared/components/auth-service/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  private checkIsLoggedIn(): void {
+    const token: string = localStorage.getItem('userData');
+    if (token) {
+      this.isLoggesIn = true;
+    }
+  }
+
+  public logIn(): void {
+    localStorage.setItem('userData', JSON.stringify({ login: 'myLogin', password: 'myPassword', token: 'myToken' }));
+
```
Comment (Yauhenia-Zhdanovich) [inline src/app/shared/components/auth-service/auth.service.ts:32]: the `isLoggesIn ` property might need to be updated on login and logout actions

File: src/app/shared/components/auth-service/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    console.log('Logged in successfully');
+  }
+
+  public logOut(): void {
+    if (localStorage.getItem('userData')) { localStorage.removeItem('userData'); }
```
Comment (Yauhenia-Zhdanovich) [inline src/app/shared/components/auth-service/auth.service.ts]: pls consider formatting `if` block

File: src/app/shared/components/auth-service/auth.service.ts (hunk 1 above)
Comment (MarharytaKrasikava) [reply src/app/shared/components/auth-service/auth.service.ts:32]: done

File: src/app/shared/components/auth-service/auth.service.ts (hunk 2 above)
Comment (MarharytaKrasikava) [reply src/app/shared/components/auth-service/auth.service.ts]: done

**Pull request merged**